lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -186,6 +187,17 @@ def _compute_block_score(
                     weight,
                 )
 
+    def _get_explicit_graded(self, block, course_structure):
+        """
+        Returns the explicit graded field value for the given block
+        """
+        field_value = course_structure.get_transformer_block_field(
+            block.location,
+            GradesTransformer,
+            GradesTransformer.EXPLICIT_GRADED_FIELD_NAME
+        )
+        return True if field_value is None else field_value
```

Comment:
Since none of the block's parent-chains explicitly denied the block from being graded, we use it in the graded computation so it's available for use, whenever needed.  This way, when the subsection's `graded` value changes, the aggregated `graded` value from its descendants is available.